### DISCUSSION TITLE
Provide Qt transport

### DISCUSSION THREAD

**Discussion by fro0m** [getsentry/sentry-native#811] [Q&A]
I have built Sentry Native for Android with Qt support. But  sentry logging does not work in Android Qt App because no transport implemented. 
I also found that some other developers [need Qt cross-platform transport](https://stackoverflow.com/questions/70573760/invalid-transport-with-sentry-for-qt-on-android).

**Comment by ashwoods** [comment]
Hi @fro0m! As the stackoverflow answer suggests, we recommend using [the sentry SDK for java and android](https://github.com/getsentry/sentry-java), which already embeds `sentry-native` as a submodule. To integrate QT, you could vendor the QT integration as described in this [issue](https://github.com/getsentry/sentry-java/issues/1626).

**Reply by fro0m** [reply]
The question is how to make it work with Qt, not with Native android and IOs.
Qt has cross-platform API for using HTTP.

**Comment by supervacuus** [comment]
@ashwoods, this should be converted to a discussion.
